report.txt

```diff
@@ -366,6 +366,7 @@ Error: Invalid selector: '+js(nano-remove-elements-onready.js, .screening)' -- '
 Error: "Unknown pseudo class: +js(insert-iframe, 1, honeypot, https://www.napiprojekt.pl/napisy-42731-Ostatni-U-Boot-(1993), width: 1px !important; height: 1px !important; position: absolute !important; left: -3000px !important)" in the rule: "napiprojekt.pl##+js(insert-iframe, 1, honeypot, https://www.napiprojekt.pl/napisy-42731-Ostatni-U-Boot-(1993), width: 1px !important; height: 1px !important; position: absolute !important; left: -3000px !important)"
 Error: Invalid selector: '+js(ac, active, #m_hot_1)' -- '+js(ac, active, #m_hot_1)' is not a valid selector in rule: ppe.pl##+js(ac, active, #m_hot_1)
 Error: Invalid selector: '+js(ac, box_active, #hot_1)' -- '+js(ac, box_active, #hot_1)' is not a valid selector in rule: ppe.pl##+js(ac, box_active, #hot_1)
+Error: Invalid selector: 'div[class]:if(>div[class]:first-child:has-text(REKLAMA):if-not(>*))' -- Error: unable to select by 'div[class]:if(>div[class]:first-child:if-not(>*))' — 'div[class]:if(>div[class]:first-child:if-not(>*))' is not a valid selector in rule: money.pl##div[class]:if(>div[class]:first-child:has-text(REKLAMA):if-not(>*))
```

Comment:
@AdguardTeam/filters-maintainers 
we have removed `:if()` and `:if-not()` pseudo-classes so the rule should be considered as invalid

but I will check and try to fix the error message later